CODEOWNERS

```diff
@@ -54,8 +54,13 @@ apps/darts-modernisation/ @hmcts/darts
 ### JUROR
 apps/juror/ @hmcts/juror
 
+<<<<<<< HEAD
+### DATA CATALOGUE (DC)
+apps/dc/ @hmcts/data-catalogue-team
+=======
 ### OPAL
 apps/opal/ @hmcts/opal
+>>>>>>> master
```

Comment:
```suggestion

### DATA CATALOGUE (DC)
apps/dc/ @hmcts/data-catalogue-team

### OPAL
apps/opal/ @hmcts/opal

```

Reply:
Sorry about this one, didn't get highlighted as a conflict